syntax = "proto3";

import 'files/common/common.proto';

message ArmoryCollections {
  repeated ArmoryCollection armory_collections = 1;

  message ArmoryCollection {
    ObjectIdentity identity = 1;
    ArmoryCollectionInfo info = 2;
  }

  message ArmoryCollectionInfo {
    string name_placeholder = 3;
    ObjectIdentity gear_set = 4;
    int32 priority = 8;
    repeated ArmoryBonus bonuses = 9;
  }

  message ArmoryBonus {
    int32 quality = 1;
    int32 quantity = 2;
    ObjectIdentity bonus_info = 3;
  }
}
